x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -20,7 +20,22 @@ export type AlertListState = Immutable<{
   alerts: AlertData[];
 }>;
 
+/**
+ * Policy list store state
+ */
+export interface PolicyListState {
+  /** Array of policy items  */
+  policyItems: object[];
```

Comment:
FYI: this type will be adjusted to one of the Ingest types that describes the data struture  (part of elastic/endpoint-app-team/issues/150 )